src/iop/ashift.c

```diff
@@ -5254,7 +5268,7 @@ static int _event_structure_auto_clicked(GtkWidget *widget, GdkEventButton *even
       g->jobparams = enhance;
     }
 
-    dt_dev_add_history_item(darktable.develop, self, TRUE, TRUE); // also calls dt_control_queue_redraw_center
+    dt_dev_add_history_item(darktable.develop, self, TRUE, FALSE); // also calls dt_control_queue_redraw_center
```

Comment:
Does the comment still apply when called with invalidate=FALSE?

Reply:
yes, only the pipe recompute is "skipped". everything else remains the same